var/spack/repos/builtin/packages/libxc/package.py

```diff
@@ -25,7 +25,7 @@
 from spack import *
 
 
-class Libxc(Package):
+class Libxc(AutotoolsPackage):
```

Comment:
this can certainly be factored as a stand-alone PR

Reply:
Done under  #7486, as you already approved – thank you.

Reply:
I'll review these changes in #7486, can you remove these from this PR?

Reply:
Absolutely.  I rolled back the `libxc` change from this branch. It will show with my next push.